Aztec/Classes/TextKit/TextView.swift

```diff
@@ -223,6 +223,10 @@ open class TextView: UITextView {
     // MARK: - Properties: Text Lists
 
     var maximumListIndentationLevels = 7
+    
+    // MARK: - Properties: Blockquotes
+    
+    var maximumBlockquoteIndentationLevels = 9
```

Comment:
This probably should be synced with the values on the LayoutManager like this:
```
var maximumBlockquoteIndentationLevels = { get { return layout. quoteMaxDepth } }
```

Reply:
well, so actually this var `maximumBlockquoteIndentationLevels` means something different than `quoteMaxDepth` 

`maximumBlockquoteIndentationLevels` means how many possible nested quote levels there could be, how many times tab keeps indenting.

`quoteMaxDepth` is how many different colors it supports before subsequent nested tabs just use the same color.

Other editors, like Froala, or apple mail app, support many quote levels, 20+, (more than I thought actually) however only 3 levels of colors. I thought supporting 5 levels of colors provided a better experience. And actually, I guess I should increase the other limit to something much higher than 9

https://www.froala.com/wysiwyg-editor